Oracle.md

```diff
@@ -198,6 +200,30 @@ The TLV serialization of oracle announcements is as follow.
 
 where `signature` is a Schnorr signature over a sha256 hash of the serialized `oracle_event`, using the tag `announcement/v0`.
 
+## Oracle Attestations
+
+After an event occurs, and the oracle creates signatures to attest the outcome, it needs to give them to users.
+An oracle can use an attestation tlv to give users this information.
+
+The TLV serialization of oracle attestations is as follows.
+
+#### Version 0 `oracle_attestation`
+
+1. type: 55400 (`oracle_attestation_v0`)
+2. data:
+    * [`string`:`event_id`]
+    * [`x_point`:`oracle_public_key`]
+    * [`u16`: `nb_signatures`]
+    * [`signature`:`signature_1`]
+    * ...
+    * [`signature`:`signature_n`]
+    * [`string`:`outcome_1`]
+    * ...
+    * [`string`:`outcome_n`]
```

Comment:
Why not just have the outcome as a single string? The respective messages for each signature can then just be inferred from the single outcome string. That seems cleaner to me so clients can figure out what happened without have to reconstruct it from the list of strings.

Reply:
For me it feels cleaner to put exactly what the oracle signed without requiring work from client to reconstruct it, but I'm not strongly opposed to your idea. Also in the (maybe unlikely) case where the oracle signs multiple unrelated strings for a single event we could then not reuse that.

Reply:
I agree with @Tibo-lg. This should be clearer and less prone to mistakes.

Reply:
I agree that this should be kept as is. TLV is a serialization format which is not meant for users to every really deal with directly, their software can handle the concatenation for them and this should be kept "protocol-level"